unity-sdk/AndroidPlugin/OptimoveUnityPlugin/build.gradle

```diff
@@ -44,4 +44,7 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
     compileOnly 'com.optimove.android:optimove-android:7.+'
     compileOnly files('libs/classes.jar')
+    compileOnly files('libs/firebase-messaging-unity-10.0.1.aar')
```

Comment:
this not needed anymore

Reply:
deleted and removed firebaseMessigng.unityPackage from the project